Tests fail on fresh checkout because of PhantomJS (OSX)

After a fresh checkout and `npm install` on the repo I can't seem to be able to run the tests for the project. Here's what I did:

```
➜  react-redux-starter-kit git:(master) git st
On branch master
Your branch is up-to-date with 'origin/master'.
nothing to commit, working directory clean
➜  react-redux-starter-kit git:(master) npm test
> react-redux-starter-kit@2.0.0-alpha.4 test /Users/ciminian/code/acme/src/react-redux-starter-kit
> better-npm-run test

running better-npm-run in /Users/ciminian/code/acme/src/react-redux-starter-kit
Executing script: test

to be executed: babel-node ./node_modules/karma/bin/karma start build/karma.conf
  app:config Create configuration. +0ms
  app:config Apply environment overrides for NODE_ENV "test". +2ms
  app:config No configuration overrides found for NODE_ENV "test" +1ms
  app:webpack:config Create configuration. +1s
  app:webpack:config Apply ExtractTextPlugin to CSS loaders. +6ms
  app:karma Create configuration. +1ms

[...]

ERROR in ./~/enzyme/build/react-compat.js
Module not found: Error: Cannot resolve module 'react/addons' in /Users/ciminian/code/acme/src/react-redux-starter-kit/node_modules/enzyme/build
 @ ./~/enzyme/build/react-compat.js 37:16-39
Child html-webpack-plugin for "index.html":
         Asset    Size  Chunks       Chunk Names
    index.html  520 kB       0
    chunk    {0} index.html 503 kB
        [0] ./~/html-webpack-plugin/lib/loader.js!./src/index.html 759 bytes {0} [built]
        [1] ./~/html-webpack-plugin/~/lodash/lodash.js 502 kB {0} [built]
        [2] (webpack)/buildin/module.js 251 bytes {0} [built]
Child extract-text-webpack-plugin:
    chunk    {0} extract-text-webpack-plugin-output-filename 2.54 kB [rendered]
        [0] ./~/css-loader?sourceMap&-minimize&modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!./~/postcss-loader!./~/sass-loader?sourceMap!./src/views/HomeView/HomeView.scss 1.03 kB {0} [built]
        [1] ./~/css-loader/lib/css-base.js 1.51 kB {0} [built]
Child extract-text-webpack-plugin:
    chunk    {0} extract-text-webpack-plugin-output-filename 22.1 kB [rendered]
        [0] ./~/css-loader?sourceMap&-minimize&modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!./~/postcss-loader!./~/sass-loader?sourceMap!./src/styles/core.scss 20.6 kB {0} [built]
        [1] ./~/css-loader/lib/css-base.js 1.51 kB {0} [built]
11 04 2016 19:33:26.780:INFO [karma]: Karma v0.13.22 server started at http://localhost:9876/
11 04 2016 19:33:26.795:INFO [launcher]: Starting browser PhantomJS
11 04 2016 19:33:28.230:INFO [PhantomJS 1.9.8 (Mac OS X 0.0.0)]: Connected on socket /#Rl_kDnqON28XgtRhAAAA with id 48381918
PhantomJS 1.9.8 (Mac OS X 0.0.0) ERROR
  SyntaxError: Parse error
  at /Users/ciminian/code/acme/src/react-redux-starter-kit/tests/test-bundler.js:55 <- webpack:///tests/test-bundler.js:6:0

Finished in 0.197 secs / 0 secs

SUMMARY:
✔ 0 tests completed
npm ERR! Test failed.  See above for more details.
```
I don't really know what's wrong here, as I'm fairly noob-ish to the stack. Any help is appreciated.

Thanks for this project - it's was really helpful for me to get started with react/redux!

Ok, I've just entered the cornfields. My project runs its tests, but _only_ if I change `karma.conf.js`:

```
➜  react-redux-starter-kit git:(master) npm test

> react-redux-starter-kit@2.0.0-alpha.3 test /Users/ciminian/code/profiler-website/src/react-redux-starter-kit
> better-npm-run test

running better-npm-run in /Users/ciminian/code/profiler-website/src/react-redux-starter-kit
Executing script: test

to be executed: babel-node ./node_modules/karma/bin/karma start build/karma.conf
  app:config Create configuration. +0ms
  app:config Apply environment overrides for NODE_ENV "test". +1ms
  app:config No configuration overrides found for NODE_ENV "test" +2ms
  app:webpack:config Create configuration. +778ms
  app:webpack:config Apply ExtractTextPlugin to CSS loaders. +6ms
  app:karma Create configuration. +1ms

START:
11 04 2016 21:09:20.578:INFO [karma]: Karma v0.13.22 server started at http://localhost:9876/
11 04 2016 21:09:20.588:INFO [launcher]: Starting browser PhantomJS
11 04 2016 21:09:21.437:INFO [PhantomJS 2.1.1 (Mac OS X 0.0.0)]: Connected on socket /#-MQM4skImTvc2mnAAAAA with id 72105283
PhantomJS 2.1.1 (Mac OS X 0.0.0) ERROR
  SyntaxError: Expected token ']'
  at /Users/ciminian/code/profiler-website/src/react-redux-starter-kit/tests/test-bundler.js:55 <- webpack:///tests/test-bundler.js:6:0

Finished in 0.487 secs / 0 secs

SUMMARY:
✔ 0 tests completed
npm ERR! Test failed.  See above for more details.

# changes to karma.conf.js
➜  react-redux-starter-kit git:(master) git df
diff --git a/build/karma.conf.js b/build/karma.conf.js
index a66d504..b8733fc 100644
--- a/build/karma.conf.js
+++ b/build/karma.conf.js
@@ -64,7 +64,7 @@ const karmaConfig = {
 if (config.coverage_enabled) {
   karmaConfig.reporters.push('coverage')
   karmaConfig.webpack.module.preLoaders = [{
-    test: /\.(js|jsx)$/,
+    test: /it_does_not_care_what_I_put_here\.(js|jsx)$/,
     include: new RegExp(config.dir_client),
     loader: 'isparta',
     exclude: /node_modules/
➜  react-redux-starter-kit git:(master) ✗ npm test
[...]
SUMMARY:
✔ 29 tests completed

=============================== Coverage summary ===============================
Statements   : 100% ( 0/0 )
Branches     : 100% ( 0/0 )
Functions    : 100% ( 0/0 )
Lines        : 100% ( 0/0 )
================================================================================
```

It doesn't seem to care what I change it to and it runs all tests, it just doesn't want the default the project comes with. This does not fix the vanilla project, that still times out.

Coverage is wrong, but if I change tests they break so it's running them. Really weird.

--- a/build/karma.conf.js
+++ b/build/karma.conf.js
@@ -47,6 +47,7 @@ const karmaConfig = {
     },
     externals: {
       ...webpackConfig.externals,
+      'react/addons': true,
       'react/lib/ExecutionEnvironment': true,
       'react/lib/ReactContext': 'window',
       'text-encoding': 'window'

